packages/itwin/map-layers/src/ui/widget/MapLayerDroppable.tsx

```diff
@@ -249,7 +242,6 @@ export function MapLayerDroppable(props: MapLayerDroppableProps) {
             activeViewport={props.activeViewport}
             mapLayerSettings={activeLayer}
             onMenuItemSelection={props.onMenuItemSelected}
-            disabled={props.disabled}
```

Comment:
why the change?

Reply:
by default the property wasn't available. I cast the component to a `div`, and readded the disable property now